api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/LRA.java

```diff
@@ -234,45 +234,55 @@
     long timeLimit() default 0;
 
     /**
-     * @return the unit of time that the {@link LRA#timeLimit()} attribute is
+     * @return the unit of time that the {@link LRA#timeLimit()} element is
      * measured in.
      */
     ChronoUnit timeUnit() default ChronoUnit.SECONDS;
 
     /**
      * <p>
-     * Normally if an LRA is present when a bean method is executed it will not
-     * be ended when the method returns. To override this behaviour and force LRA
-     * termination on exit use the end element.
+     * Normally if an LRA is present when a resource method is executed it will not
+     * be ended when the method returns. To override this behaviour and force the LRA
+     * to close on exit use this <code>end</code> element with the value set to
+     * {@literal true}.
```

Comment:
The default value is 'true'. So normally (without explicitly changing the value to false) LRA is ended when the method returns.

Reply:
okay

Reply:
@rdebusscher I have just pushed a commit that makes sure the documentation around `end` is accurate.